### ISSUE TITLE
single hand considerations for large screen phones

### ISSUE THREAD

**Issue by joopdo** [jkuester/unlauncher#213]
I just moved to a larger screen phone, and find myself unable to open an application without falling back to two hands.

Some ideas, curious if you considered them before?

- when I search for an application name, one there is only one result remaining, could the application open when I hit <enter> on the keyboard?
- instead of making the text tappable, consider the entire horizontal block to be tappable (easier for right handed people)
- moving search bar result list down. This one might be more complicated. To put the results within reach on larger phone screens.

Curious what you think and thanks for this launcher!

**Comment by jkuester** [comment]
Good news!  The upcoming https://github.com/jkuester/unlauncher/releases/tag/2.1.1 version should address almost all of these considerations!  

- :heavy_check_mark: Hitting enter when searching will opens the top filtered app
- :heavy_check_mark: Pressing anywhere within the whole line of an app in the app list should open the app (additionally, we have added the option to change the alignment of the apps in the list to the right-hand side or even the center).
- For the search bar, we have added an option to move the search box to the bottom of the screen instead of the top. However, currently the list of filtered apps remains aligned to the top. I am open to thoughts and feedback from everyone on if it would be better to align it with the bottom when the search bar is moved down!  

The new version with these updates should be published on F-Droid in the next couple of days.  Or, the `2.1.0-beta.1` version  is already available to be manually installed from F-Droid if you want to get it sooner!  (We did not find any bugs with the beta, so it should be as stable as `2.1.1`.)
